### DISCUSSION TITLE
Dark mode for browser protected websites?

### DISCUSSION THREAD

**Discussion by bnaman50** [darkreader/darkreader#8396] [Q&A] [answered]
Hello, 

I was wondering if there is any way to Invert the Browser protected sites? It literally hurts the eyes when your entire system is in dark mode and suddenly a white website pops up 😅

Here is what I'm talking about. 

<img width="1142" alt="image" src="https://user-images.githubusercontent.com/5251592/157286778-853a47ea-0995-4a1b-affb-a497ac87ff18.png">

As you can see, dark mode is not working on this website even though it is in the Invert list. Just wondering if I could do something about it. 

Thanks, 
Naman

**Comment by Gusted** [comment, marked as answer]
They are protected websites for a reason, defeats the purpose if you can disable that feature.

**Comment by bnaman50** [comment]
Thanks for your quick answer. I was simply hoping someone had found a loophole there 😅
